android/src/main/java/com/horcrux/svg/SvgView.java

```diff
@@ -361,15 +363,25 @@ String toDataURL() {
     return Base64.encodeToString(bitmapBytes, Base64.NO_WRAP);
   }
 
-  String toDataURL(int width, int height) {
+  String toDataURL(final ReadableMap options) {
+    int width = options.hasKey("width") ? options.getInt("width") : getWidth();
+    int height = options.hasKey("height") ? options.getInt("height") : getHeight();
```

Comment:
Are you sure it's still working after moving the width/height to `size` property?

Reply:
Already fixed, that exception.